backend/tests/unit/dal/Services/Admin/PropertyClassificationTest.cs

```diff
@@ -0,0 +1,128 @@
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Pims.Core.Comparers;
+using Pims.Core.Extensions;
+using Pims.Core.Test;
+using Pims.Dal.Entities.Models;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Pims.Dal.Services.Admin;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Xunit;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Dal.Test.Services.Admin
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("area", "admin")]
+    [Trait("group", "propertyclassification")]
+    [ExcludeFromCodeCoverage]
+    public class PropertyClassificationServiceTest
+    {
+        #region Tests
+        #region Get
+        [Fact]
+        public void GetAll_PropertyType()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission(Permissions.SystemAdmin);
+
+            using var init = helper.InitializeDatabase(user);
+
+            var service = helper.CreateService<PropertyClassificationService>(user);
+            var context = helper.GetService<PimsContext>();
+
+            // Act
+            var result = service.GetAll();
+
+            // Assert
+            // There are 8 classificaitons by default
+            Assert.True(result.Count() > 0);
```

Comment:
Not a great test lol

Reply:
can probably compare against the number of classifications in the context.